packages/design-preview/src/components/index.tsx

```diff
@@ -19,40 +26,68 @@ interface PreviewProps {
 	onClickCategory?: ( category: Category ) => void;
 	actionButtons: React.ReactNode;
 	recordDeviceClick: ( device: string ) => void;
+	siteId: number;
+	stylesheet: string;
+	selectedFontVariation: GlobalStylesObject | null;
+	onSelectFontVariation: ( variation: GlobalStylesObject | null ) => void;
+	onGlobalStylesChange: ( globalStyles: GlobalStylesObject | null ) => void;
 }
 
 const INJECTED_CSS = `body{ transition: background-color 0.2s linear, color 0.2s linear; };`;
 
 const getVariationBySlug = ( variations: StyleVariation[], slug: string ) =>
 	variations.find( ( variation ) => variation.slug === slug );
 
-const Preview: React.FC< PreviewProps > = ( {
+// @todo Get the style variations of theme, and then combine the selected one with colors & fonts for consistency
```

Comment:
@arthur791004 Can you explain more about this `@todo`? I'm interested (maybe add this in the follow-up task section in this PR description)

Reply:
I see, super clear, thanks for the explanation! 😄